.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
@ipedrazas My apologies. You can refer to Makefile of Gorush project. https://github.com/appleboy/gorush/blob/master/Makefile#L22-L26

Your config is correct if you don't have check `DRONE_TAG` variable on Makefile or something else.